docs/cow-protocol/tutorials/solvers/flashloans_support.md

```diff
@@ -0,0 +1,77 @@
+---
+sidebar_position: 4
+---
+
+# How to execute flashloans orders
+
+This short tutorial explains how orders with flashloans hints can be executed by solvers.
+
+## Where to get information about an order’s flashloan?
+
+This information is added by the user inside the order’s appdata. If you have a colocated driver you’ll only get the appdata hash as part of the order from the autopilot.
+
+For example this [order](https://explorer.cow.fi/gc/orders/0x413a7246f58441ad92ea19c09cef90d1b23a1e211e0963f3d39b7db48140533d669685c660c260d80b614f8d1a5ffd24c4e3b82668cd8760) (staging) has the appdata hash `0xa8476d34ec1e22b818cf3d6289e30271a04a4e38a8a3a71f947b5881de06a852` which can be looked up [here](https://barn.api.cow.fi/xdai/api/v1/app_data/0xa8476d34ec1e22b818cf3d6289e30271a04a4e38a8a3a71f947b5881de06a852).
+
+As you can see the `flashloan` field contains just a few pieces of information:
+- `amount`: units (in wei) that should be flashloaned
+- `token`: which token should be flashloaned
+- `receiver`: address that is supposed to receive the tokens
+- `liquidityProvider`: the lending protocol’s contract (i.e. AavePool)
+- `protocolAdapter`: which address needs to be used to adapt the `liquidityProvider` to CoW Protocol’s flashloan machinery.
+
+The reference driver is already doing all that for you and is also propagating the flashloan data to the connected solver endpoint as a field on the `order` struct. For example, [this](https://solver-instances.s3.eu-central-1.amazonaws.com/staging/xdai/auction/146823591.json) is a `/solve` request that your solver will get from the driver (just search for `flashloan` and you will find it).
+
+## Adjustments needed in the solver/driver
+
+### Encode the Call to the Flashloan Router
+
+Because the flashloan needs to be available over the entire duration of the settlement, all flashloan settlements will have to go through the [flashloan router](https://docs.cow.fi/cow-protocol/reference/contracts/periphery/flashloans#iflashloanrouter-contract) (deterministically deployed at `0x9da8b48441583a2b93e2ef8213aad0ec0b392c69`). So instead of calling `settle()` on the settlement contract you have to call `flashloanAndSettle()` on the flashloan router contract. This is straightforward, and only takes 2 arguments:
```

Comment:
maybe `0x9da8b48441583a2b93e2ef8213aad0ec0b392c69` should be a link, so people can see the contract easily

Reply:
Added a short sentence to reference the mainnet deployment, and provided an etherscan link